src/test/java/qna/domain/AnswerTest.java

```diff
@@ -2,21 +2,48 @@
 
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import qna.NotFoundException;
+import qna.UnAuthorizedException;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.*;
 import static org.junit.jupiter.api.Assertions.assertAll;
 
 public class AnswerTest {
     public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
     public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
 
+    @DisplayName("Answer 도메인 생성")
+    @Test
+    void test_new() {
+        //given & when
+        Answer answer = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents");
+        //then
+        assertThat(answer).isNotNull();
+    }
+
+    @DisplayName("Answer 도메인 생성 시 Question 이 null 이면 NotFoundException")
```

Comment:
만약 Exception 이 변경되게 되면 테스트 설명도 수정을 해줘야 할 것 같아요.
테스트 코드 또한 유지 보수 비용이 들어가기 때문에 최대한 유지보수 하기 좋게 작성하는게 좋은데요,
테스트 설명을 구현에 대해 적어주기 보다는 요구사항 중심으로 작성해 보면 어떨까요? 😄 
```suggestion
    @DisplayName("작성자가 없으면 Answer를 생성할 수 없다.")
```